daemon/daemon_linux_test.go

```diff
@@ -228,3 +232,75 @@ func TestShouldUnmountRoot(t *testing.T) {
 		})
 	}
 }
+
+func checkMounted(t *testing.T, p string, expect bool) {
+	t.Helper()
+	mounted, err := mount.Mounted(p)
+	assert.Check(t, err)
+	assert.Check(t, mounted == expect, "expected %v, actual %v", expect, mounted)
```

Comment:
nit: doesn't `assert.Check()` already output a readable message if the comparison fails?

Reply:
nope